services/consensusalgo/benchmarkconsensus/leader.go

```diff
@@ -173,7 +163,7 @@ func (s *service) leaderHandleCommittedVote(sender *gossipmessages.SenderSignatu
 	// validate the vote
 	err := s.leaderValidateVoteUnderMutex(sender, status)
 	if err != nil {
-		s.reporting.Error(err.Error()) // TODO: wrap with added context
+		s.reporting.Error("leader failed to validate vote", instrumentation.Error(err))
 		return
```

Comment:
this logs the error but then does not return it to the caller, is that the intended behavior?

Reply:
yes, the caller is a gossip incoming message handler.. it doesn't have anything to do with the error. this is the top level so it should log and ignore on problem